airbyte-webapp/src/views/Connector/ConnectorForm/components/FrequentlyUsedConnectors/FrequentlyUsedConnectorsCard.test.tsx

```diff
@@ -2,14 +2,17 @@ import { fireEvent, render, waitFor } from "@testing-library/react";
 import { IntlProvider } from "react-intl";
 import { mockDestinationsData } from "test-utils/mock-data/mockFrequentlyUsedDestinations";
 
+import { defaultOssFeatures, FeatureService } from "hooks/services/Feature";
 import en from "locales/en.json";
 
 import { FrequentlyUsedConnectorsCard, FrequentlyUsedConnectorsCardProps } from "./FrequentlyUsedConnectorsCard";
 
 const renderFrequentlyUsedConnectorsComponent = (props: FrequentlyUsedConnectorsCardProps) =>
   render(
     <IntlProvider locale="en" messages={en}>
-      <FrequentlyUsedConnectorsCard {...props} />
+      <FeatureService features={defaultOssFeatures}>
+        <FrequentlyUsedConnectorsCard {...props} />
+      </FeatureService>
```

Comment:
We also have `src/test-utils/tesutils.tsx` that exports its own `render()` which wraps the test component with `IntlProvider` and `FeatureService`, just FYI!

Reply:
Ooh, good tip. I'll make that change before merging, to save the next person from the ol' "test context is different from app context" surprise failure.